### ISSUE TITLE
Unable to find button having \n in accessibilityLabel

### ISSUE THREAD

**Issue by onkarbapat** [calabash/calabash-ios#92]
I ran following query on my app
  query "Button", :accessibilityLabel

one of the labels is "Manage\nItems"

If I use the same label name in the cucumber .feature file like this
- I should see "Manage\nItems"

calabash is unable to validate the existence of the button.
